package/netsurf/package

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+# Copyright (c) 2021 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(netsurf)
+pkgdesc="Lightweight web browser"
+url=https://github.com/alex0809/netsurf-reMarkable
+pkgver=0.3.3-1
+timestamp=2021-04-12T15:14+00:00
+maintainer="Alex Friesenhahn <rm-dev@alexfriesenhahn.de>"
+archs=(rmall)
+license=MIT
+section="readers"
+installdepends=(
+    dejavu-fonts-ttf-DejaVuSans
+    dejavu-fonts-ttf-DejaVuSans-Bold
+    dejavu-fonts-ttf-DejaVuSans-BoldOblique
+    dejavu-fonts-ttf-DejaVuSans-Oblique
+    dejavu-fonts-ttf-DejaVuSerif
+    dejavu-fonts-ttf-DejaVuSerif-Bold
+    dejavu-fonts-ttf-DejaVuSerif-Italic
+    dejavu-fonts-ttf-DejaVuSansMono
+    dejavu-fonts-ttf-DejaVuSansMono-Bold
+)
+image=base:v1.5
+
+source=(
+    https://github.com/alex0809/netsurf-reMarkable/archive/refs/tags/v0.3.3.tar.gz
+    netsurf.draft
+)
+sha256sums=(
+    9d6bb15af25604a390a45d665445b53b8b2530f01240054dbe1a7c306f1b90ac
+    SKIP
+)
+
+build() {
+    scripts/install_dependencies.sh
+    TARGET_WORKSPACE=$(pwd)/build scripts/build.sh
+
+    # After build, the Netsurf directory contains some broken symlinks, delete them before continuing
```

Comment:
What were these symlinks for?

Reply:
Something entirely unrelated, some links in the GTK and RISC OS frontends.

```
./build/netsurf/frontends/gtk/res/Messages
./build/netsurf/frontends/riscos/appdir/Resources/en/Messages
./build/netsurf/frontends/riscos/appdir/Resources/nl/Messages
./build/netsurf/frontends/riscos/appdir/Resources/it/Messages
./build/netsurf/frontends/riscos/appdir/Resources/de/Messages
./build/netsurf/frontends/riscos/appdir/Resources/fr/Messages
```
I haven't touched these frontends at all.

Reply:
I just had a closer look and those symlinks are already broken in the git repository. So I'll just remove them from there entirely.